$OpenBSD: patch-khelpcenter_CMakeLists_txt,v 1.1 2014/01/10 08:12:11 zhuk Exp $
Deconflict with kdebase-3.x. See also patches in dev-scripts,
kde-workspace and kdelibs.
--- khelpcenter/CMakeLists.txt.orig	Sat Dec 28 20:30:06 2013
+++ khelpcenter/CMakeLists.txt	Sat Dec 28 20:30:36 2013
@@ -48,6 +48,8 @@ kde4_add_kcfg_files(khelpcenter_KDEINIT_SRCS prefs.kcf
 kde4_add_app_icon(khelpcenter_KDEINIT_SRCS "${KDE4_INSTALL_DIR}/share/icons/oxygen/*/apps/help-browser.png")
 
 kde4_add_kdeinit_executable( khelpcenter ${khelpcenter_KDEINIT_SRCS})
+set_target_properties(khelpcenter PROPERTIES
+	OUTPUT_NAME khelpcenter4)
 target_link_libraries(kdeinit_khelpcenter  ${KDE4_KHTML_LIBS}  ${QT_QTXML_LIBRARY} ${KDE4_KCMUTILS_LIBS} )
 install(TARGETS kdeinit_khelpcenter  ${INSTALL_TARGETS_DEFAULT_ARGS} )
 install(TARGETS khelpcenter ${INSTALL_TARGETS_DEFAULT_ARGS})
